src/utils/displayStrings.ts

```diff
@@ -0,0 +1,36 @@
+export const EROOR_MESSAGE = {
+  ROOM_NOT_FOUND:
+    'The room has not been started. If your legal session is due to start please wait a minute and try again',
+  INVALIDE_CREDENTIALS: '. Please make sure you are using the correct credentials.',
+  UNSUPPORTED_MESSAGE: 'This browser is not supported. Please change/upgrade your browser.',
+  NETWORK_ERROR: 'Network Error',
+};
+
+export const PLAYER_STATE = {
+  stopped: 'stopped',
+};
+
+export const ROOMSTATE = {
+  RECONNECTING: 'reconnecting',
+  RE_CONNECTED: 'reconnected',
+  DISCONNECTED: 'disconnected',
+  CONNECTED: 'connected',
```

Comment:
Are all those displayStrings or some of them are for internal usage? If so, I would separate them to two files, with different object / variable suffix.